test/unit/math/prim/scal/fun/distance_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stan/math/prim/scal.hpp>
+#include <gtest/gtest.h>
+
+TEST(MathFunctions, distance) {
+  double x1 = 1;
+  double x2 = 4;
+
```

Comment:
Should we also check proper throws, and messages for incorrect inputs (NaN, Inf)? It's done in the test for `cov_periodic` (`gp_exp_quad_cov)`, but not in `distance_test.cpp`.